### DISCUSSION TITLE
Please add a filter on the search bar.

### DISCUSSION THREAD

**Discussion by Redbeardjunior** [dailydotdev/daily#991] [Feature Requests]
I'd like to request the addition of a filter for the search bar. This filter should allow us to search for recommended posts or sort posts by their publication date, whether older or more recent.

**Comment by idoshamun** [comment]
Hi @Redbeardjunior,

We have a discussion going on this suggestion. Feel free to join: https://github.com/dailydotdev/daily/discussions/294
